kervintznoel-2026/portfolio/styles/globals.css

```
@@ -0,0 +1,91 @@
+  --color-subtle:     #444444;
+
+  /* Fonts */
+  --font-display: 'DM Serif Display', Georgia, serif;
+  --font-body:    'Inter', system-ui, sans-serif;
+  --font-mono:    'DM Mono', monospace;
+
+  /* Animations */
+  --animate-fade-up: fadeUp 0.6s ease forwards;
+  --animate-fade-in: fadeIn 0.4s ease forwards;
+
+  @keyframes fadeUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)